docs/running-a-node/docker-management.md

```diff
@@ -140,6 +140,31 @@ Once modifications are complete, [start up the docker container again](#starting
 
 ---
 
+### Docker entrypoint support
+
+The docker entry script has migrated to a command with default arguments:
+```
+Usage:
+   /entry.sh nano_node [[--]daemon] [cli_options] [-l] [-v size]
+     [--]daemon
+       start as daemon either cli [--daemon] form or short form [daemon]
+     cli_options
+       nano_node cli options <see nano_node --help>
+     -l
+       log to console <use docker logs {container}>
+     -v<size>
+       vacuum database if over size GB on startup
+   /entry.sh bash [other]
+     other
+       bash pass through
+   /entry.sh [*]
+     *
+       usage
```

Comment:
What does this mean?

Reply:
If you pass something in that doesnt begin with nano_node or bash

Reply:
And usage is just indicating it will return a usage script